test/unit/serialize.js

```diff
@@ -424,6 +424,29 @@ describe('serialize( obj )', function () {
             expect(serialize(["<"], {space: 2})).to.equal('[\n  "\\u003C"\n]');
             expect(serialize(["<"], {unsafe: true, space: 2})).to.equal('[\n  "<"\n]');
         });
+
+        it("should accept a `ignoreFunction` option", function() {
+            function fn() { return true; }
+            const obj = {
+              fn,
+              fn_arrow: () => {
+                return true;
+              }
+            };            
+            const obj2 = {
+                num:123,
+                str:'str',
+                fn
```

Comment:
I prefer non-shorthands in this tests code for consistency.
```suggestion
                fn: fn
```

Reply:
Agreed